numpy/core/tests/test_function_base.py

```diff
@@ -407,3 +407,29 @@ def test_round_negative(self):
         y = linspace(-1, 3, num=8, dtype=int)
         t = array([-1, -1, 0, 0, 1, 1, 2, 3], dtype=int)
         assert_array_equal(y, t)
+
+
+class TestArange:
+    def test_require_range(self):
+        import pytest
```

Comment:
The pytest import should be top-level, if it is not already.

(I am a bit surprised by the codecov comments, since those lines seem covered by these tests...)